frank/radial_fitters.py

```diff
@@ -692,6 +694,9 @@ def fit(self, u, v, V, weights=1, verbose=False):
         # Fit geometry if needed
         self._geometry.fit(u, v, V, weights)
 
+        logging.info('  Fitting for brightness profile')
+        t1 = time.time()
+
         # Project the data to the signal space
         self._build_matrices(u, v, V, weights)
```

Comment:
For consistency, I think build matrices should be logged, it can be an expensive function call - equiavelent to ~ 100 iterations, which are logged.

Reply:
If not using `time` in radial_fitters.py, this would require modifying the syntax to call `FrankFitter` so that `_build_matrices` could be called separately. I'd prefer to avoid this.

Reply:
I don't quite understand? I'm just suggesting add an extra logging message inside _build_matrices

Reply:
I've added a commit showing you want I mean

Reply:
I kept the print statement that the matrices are being built, removed `time` calls

Reply:
I thought it made sense to leave line 697 in the fit method. Do you agree?

Reply:
Now that almost all logging statements are back in fit.py, I think it's consistent to have it in fit.py, not radial_fitters.py.

Reply:
The reason I disagree is that its currently possible/common to get internal logging messages from inside the .fit method without reporting that your actually doing the fit. Surely that doesn't make sense?

Reply:
As you said, if you're using frank as a module, you'll have to call FrankFitter directly, so you'll know you're running a fit.

Reply:
It's the same logging behavior as in geometry.py

Reply:
It's not though is it? FitGeometryGaussian always gives a message that it's doing a Gaussian fit to the gemetry

Reply:
It does, but FixedGeometry doesn't print anything, so if you passed that when using the code as a module, you wouldn't get a notification the visibilities are being deprojected.

Similarly in io.py you get a notification that the UVTables are being saved, but the notification that the fit results are being saved is in fit.py as you requested.

In all 3 cases the behavior should be the same, either all 'parent' notifications in fit.py, or in the respective scripts.